samza-core/src/main/java/org/apache/samza/zk/ZkUtils.java

```diff
@@ -136,6 +137,44 @@ public synchronized String getEphemeralPath() {
     return children;
   }
 
+  /**
+   * Method is used to read processors PIDs
+   *
+   * @return absolute ZK node path
+   */
+  public String getProcessorsPID(String fullPath) {
+    String data = zkClient.<String>readData(fullPath, true);
+    if (data == null) {
+      throw new SamzaException(String.format("Cannot read ZK node:", fullPath));
+    }
+    // data format is "host pid"
+    String [] pidHost = data.split(" ");
+    return pidHost[1];
+  }
+
+    /**
+     * Method is used to get the <i>sorted</i> list of currently active/registered processors PIDs
+     *
+     * @return List of absolute ZK node paths
+     */
+  public List<String> getSortedActiveProcessorsPIDs() {
+    String processorPath = keyBuilder.getProcessorsPath();
+    List<String> children = zkClient.getChildren(processorPath);
+    List<String> childrenPids = new ArrayList<>(children.size());
```

Comment:
same comment as above. Keeping the variable name to be the same as the terms used in documentation makes the code more readable. ChildrenPids should simply be processorIds, right ?